include/nbl/video/SPhysicalDeviceLimits.h

```diff
@@ -0,0 +1,497 @@
+    //uint32_t           maxVertexAttribDivisor;
+
+    /* AccelerationStructurePropertiesKHR *//* provided by VK_KHR_acceleration_structure */
+    uint64_t           maxGeometryCount;
+    uint64_t           maxInstanceCount;
+    uint64_t           maxPrimitiveCount;
+    uint32_t           maxPerStageDescriptorAccelerationStructures;
+    uint32_t           maxPerStageDescriptorUpdateAfterBindAccelerationStructures;
+    uint32_t           maxDescriptorSetAccelerationStructures;
+    uint32_t           maxDescriptorSetUpdateAfterBindAccelerationStructures;
+    uint32_t           minAccelerationStructureScratchOffsetAlignment;
+ 
+    /* DepthStencilResolvePropertiesKHR ---> MOVED TO Vulkan 1.2 Core  */
+    /* DriverPropertiesKHR ---> MOVED TO Vulkan 1.2 Core  */
+    /* VK_KHR_fragment_shader_barycentric --> Coverage 0% --> no structs defined anywhere in vulkan headers */
+
+    /* FragmentShadingRatePropertiesKHR *//* provided by VK_KHR_fragment_shading_rate */
+    //VkExtent2D               minFragmentShadingRateAttachmentTexelSize;
+    //VkExtent2D               maxFragmentShadingRateAttachmentTexelSize;
+    //uint32_t                 maxFragmentShadingRateAttachmentTexelSizeAspectRatio;
+    //VkBool32                 primitiveFragmentShadingRateWithMultipleViewports;
+    //VkBool32                 layeredShadingRateAttachments;
+    //VkBool32                 fragmentShadingRateNonTrivialCombinerOps;
+    //VkExtent2D               maxFragmentSize;
+    //uint32_t                 maxFragmentSizeAspectRatio;
+    //uint32_t                 maxFragmentShadingRateCoverageSamples;
+    //VkSampleCountFlagBits    maxFragmentShadingRateRasterizationSamples;
+    //VkBool32                 fragmentShadingRateWithShaderDepthStencilWrites;
+    //VkBool32                 fragmentShadingRateWithSampleMask;
+    //VkBool32                 fragmentShadingRateWithShaderSampleMask;
+    //VkBool32                 fragmentShadingRateWithConservativeRasterization;
+    //VkBool32                 fragmentShadingRateWithFragmentShaderInterlock;
+    //VkBool32                 fragmentShadingRateWithCustomSampleLocations;
+    //VkBool32                 fragmentShadingRateStrictMultiplyCombiner;
+    
+    /* Maintenance2PropertiesKHR *//* provided by VK_KHR_maintenance2 *//* MOVED TO Vulkan 1.1 Core  */
+    /* Maintenance3PropertiesKHR *//* provided by VK_KHR_maintenance3 *//* MOVED TO Vulkan 1.1 Core  */
+    /* Maintenance4PropertiesKHR *//* provided by VK_KHR_maintenance4 *//* MOVED TO Vulkan 1.3 Core  */
+    /* MultiviewPropertiesKHR    *//* provided by VK_KHR_multiview    *//* MOVED TO Vulkan 1.1 Core  */
+
+    /* PerformanceQueryPropertiesKHR *//* provided by VK_KHR_performance_query */
+    // ! We don't support PerformanceQueries at the moment;
+    // ! But we have a bool with the same name in SFeatures and that is mostly for GL when NBL_ARB_query_buffer_object is reported and that holds for every query 
```

Comment:
its different, IIRC its about whether quere results can be touched by a different queue (which they can't in GL & GLES), for example when submitting a commandbuffer which copies query result to a buffer of a query which has completed as a result of a submission of a commandbuffer to another queue

`allowCommandBufferQueryCopies` is whether you can copy the PERFORMANCE query to a buffer at all (will be true for GL if we ever implement performance queries there)

Reply:
I have no idea why but Vulkan had put this as properties instead of features. Probably because it doesn't need enablement or anything for the logical device

Reply:
> Probably because it doesn't need enablement or anything for the logical device

ah, okay, yes then both should be in `Limits` then